FilePersistence/src/version_control/merge/ListMergeComponentV2.cpp

```diff
@@ -70,6 +70,42 @@ void ListMergeComponentV2::run(MergeData& mergeData)
 	}
 }
 
+QList<Model::NodeIdType> ListMergeComponentV2::computeListsToMerge(MergeData& mergeData)
+{
+	QList<Model::NodeIdType> listsToMerge;
+	auto changes = mergeData.cg_.changes_;
+	// Add every list whose child structure changes
+	for (auto change : changes)
+	{
+		// Ignore Lists with stationary changes
```

Comment:
remove Lists. You're not ignoring the lists, but changes to items of the list, if those changes are stationary.